web/react/components/user_settings/manage_incoming_hooks.jsx

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) 2015 Spinpunch, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+var Client = require('../../utils/client.jsx');
+var Utils = require('../../utils/utils.jsx');
+var Constants = require('../../utils/constants.jsx');
+var ChannelStore = require('../../stores/channel_store.jsx');
+var LoadingScreen = require('../loading_screen.jsx');
+
+export default class ManageIncomingHooks extends React.Component {
+    constructor() {
+        super();
+
+        this.getHooks = this.getHooks.bind(this);
+        this.addNewHook = this.addNewHook.bind(this);
+        this.updateChannelId = this.updateChannelId.bind(this);
+
+        this.state = {hooks: [], channelId: ChannelStore.getByName(Constants.DEFAULT_CHANNEL).id, getHooksComplete: false};
+    }
+    componentDidMount() {
+        this.getHooks();
+    }
+    addNewHook() {
+        let hook = {}; //eslint-disable-line prefer-const
```

Comment:
Why aren't we just using const here? I know ES6's const is a bit of a misnomer because it behaves more like final in Java than const in C, but it seems weird to be disabling errors when they could be fixed easily to get through eslint.